src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -1581,8 +1582,8 @@ async def delete(self, attachments):
         await self._service_change('iscsitarget', 'reload')
 
         # For SCALE, reload action will remove existing LUN(s)
```

Comment:
Create dataset on SCALE, create zvol there, mount it via iSCSI, then delete dataset. Won't you get EBUSY because reload does not happen immediately?

Reply:
No ?

Reply:
When I delete the dataset, the reload action removes the LUN and the zvol is deleted without any issue.

Reply:
It would be probably because the `scstadmin` command actually waits for the action to complete ?

Reply:
Then what does "For SCALE, reload action will remove existing LUN(s)" mean if the one that is really doing the job is `scstadmin`?

Reply:
So `iscsitarget` service reload action has been overloaded to use `scstadmin`, which means that reload of `iscsitarget` will result in LUN being removed via `scstadmin`. Please let me know if you think we should remove that comment ?